go.mod

```diff
@@ -3,6 +3,12 @@ module github.com/kansaslabs/rossby
 go 1.13
 
 require (
+	github.com/golang/protobuf v1.3.2
 	github.com/stretchr/testify v1.4.0
-	github.com/urfave/cli v1.22.1 // indirect
+	github.com/urfave/cli v1.22.1
+	golang.org/x/net v0.0.0-20191003171128-d98b1b443823
+	golang.org/x/sys v0.0.0-20191007154456-ef33b2fb2c41 // indirect
+	golang.org/x/text v0.3.2 // indirect
+	google.golang.org/genproto v0.0.0-20191002211648-c459b9ce5143 // indirect
```

Comment:
This is so slick! I'm looking forward to getting into the go modules stuff &mdash; is the `// indirect` flag something it adds automatically?

Reply:
Yes, I believe it refers to sub-dependencies of the primary dependencies. But I'm not 100% sure. This whole file was created automatically, I haven't gotten into how to update it manually - right now I'm preferring to just use `go mod`.